src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -67,3 +77,6 @@
   - elsif !@project.is_locked?
     = render partial: 'webui2/webui/request/add_role_request_dialog', locals: { project: @project }
     = render partial: 'webui2/webui/request/delete_request_dialog', locals: { project: @project }
+
+= content_for :ready_function do
+  initializeDataTable('#packages-table,#inherited-packages-table');
```

Comment:
It's a small thing, but please add a whitespace here -> `'#packages-table, #inherited-packages-table'`